apps/sms/test/marionette/lib/messages.js

```diff
@@ -64,11 +64,20 @@ var ParticipantsAccessor = require('./views/participants/accessors');
 
         Participants: new ParticipantsAccessor(client),
 
-        launch() {
+        launch(doNotWaitForAppToBecomeReady) {
           client.switchToFrame();
           client.apps.launch(ORIGIN_URL);
           client.apps.switchToApp(ORIGIN_URL);
-          client.helper.waitForElement(SELECTORS.appReady);
+
+          // In 99.99% cases we'd like to wait for the app to become fully ready
```

Comment:
App is marked as ready on "fully-loaded" and in one of the inbox tests we want to check that use can enter Composer or Conversation _before_ all conversation are loaded (we had a bug when we waited for the ```load``` event before allowing user to navigate further, but because of a big number of images user had to wait for fully loaded). See https://bugzilla.mozilla.org/show_bug.cgi?id=1217859